src/java.desktop/windows/classes/sun/awt/Win32FontManager.java

```diff
@@ -259,22 +243,15 @@ public static void registerJREFontsForPrinting() {
             pathName = fontsForPrinting;
             fontsForPrinting = null;
         }
-        java.security.AccessController.doPrivileged(
-            new java.security.PrivilegedAction<Object>() {
-                public Object run() {
```

Comment:
Is the `@SuppressWarnings("removal")` on this method still needed?

Reply:
removed